lib.sh

```diff
@@ -414,18 +416,29 @@ download_geoip_mmdb() {
     echo "GEOIPUPDATE_VERBOSE=1"
     } > /tmp/dockerenv
     unset MwKfcYATm43NMT
-    unset i9HL69SLnp4ymy 
+    unset i9HL69SLnp4ymy
     install_docker
-    if docker run --name maxmind --env-file /tmp/dockerenv -v /usr/share/GeoIP:/usr/share/GeoIP ghcr.io/maxmind/geoipupdate
+    if docker run --name maxmind --env-file /tmp/dockerenv -v "$GEOBLOCK_DIR":"$GEOBLOCK_DIR" ghcr.io/maxmind/geoipupdate
     then
         docker rm -f maxmind
         rm -f /tmp/dockerenv
+        # Since only one mmdb file can exist at the same time due to Apache "if" confitions, remove IPInfos config
+        rm -f "$GEOBLOCK_DIR"/IPInfo-Country.mmdb
     else
         docker rm -f maxmind
         rm -f /tmp/dockerenv
-        msg_box "Update limit for Maxmind GeoDatabase reached! Please try again tomorrow."
-        return 1
+        print_text_in_color "$ICyan" "Rate limit for Maxmind GeoDatabase reached! We're now trying to get the Country Database from https://ipinfo.io instead."
```

Comment:
@abdullahdevrel This is what I need approval on.

Reply:
I'll merge this now, please come back with a suggestion if the above is not enough.